### PR TITLE
Fixes #14697: Strip rust release binaries

### PR DESCRIPTION
**Pull request by amousset**

### CODE REVIEW

File: rust.makefile [hunk 1]
```diff
@@ -35,7 +35,9 @@ version:
 	rustc --version
 
 build: version
-	cargo build --release
+	# strip release binaries, cf. https://github.com/rust-lang/cargo/issues/3483#issuecomment-431209957
+	# will be configurable in Cargo.toml in te future https://github.com/rust-lang/cargo/issues/3483#issuecomment-631584439
+	RUSTFLAGS="-C link-arg=-s" cargo build --release
```
Comment (peckpeck) [inline rust.makefile:40]: could you use the long version of arguments ?

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**